STATUS: INLINE SUCCESSFUL CANREMOVE
MODIFIED: de.strullerbaumann.visualee.filter.boundary.FilterContainerTest.testIsOkExcludeFalse
MODIFIED FILE: /home/alex/Repositories/visualee/src/test/java/de/strullerbaumann/visualee/filter/boundary/FilterContainerTest.java
CLEANER: de.strullerbaumann.visualee.filter.boundary.FilterContainerTest.testAdd
CLEANER FILE: /home/alex/Repositories/visualee/src/test/java/de/strullerbaumann/visualee/filter/boundary/FilterContainerTest.java
POLLUTER: de.strullerbaumann.visualee.filter.boundary.FilterContainerTest.testIsOkExcludeTrue
POLLUTER FILE: /home/alex/Repositories/visualee/src/test/java/de/strullerbaumann/visualee/filter/boundary/FilterContainerTest.java
ORIGINAL CLEANER SIZE: 7
NEW CLEANER SIZE: 1
ELAPSED TIME: 16.461999893188477
==========================
@@ -103,0 +103,3 @@
+ {
+     FilterContainer.getInstance().clear();
+ }
